src/components/FileImport.js

```diff
@@ -54,14 +54,27 @@ class FileImport extends Nimiq.Observable {
     }
 
     /**
-     * @param {DOMEvent} event
+     * @param {Event} event
      */
     _onFileSelected(event) {
-        this.$errorMessage.textContent = '';
-        // @ts-ignore (Property 'files' does not exist on type 'EventTarget & Element'.)
-        const files = event.target.files;
-        this._readFile(files[0]);
-        this.$fileInput.value = '';
+        // @ts-ignore
```

Comment:
You could do
```suggestion
     const eventTarget = /** @type {HTMLInputElement} */(event.target);
```
and use `eventTarget` afterwards.

Reply:
that code is likely subject to change in the upcoming Login File PR. But will absolutely change them to get rid of the `@ts-ignore`.